app/views/splash/index.html.erb

```diff
@@ -1,6 +1,12 @@
 <div class="splash-container">
    <h1 class="splash-title">BUDGETARY APP TRANSACTIONS</h1>
+   <div class ="sign-in">
+  <%= link_to "Sign in", new_user_session_path, method: :delete, class:"nav-link1" %>
+
```

Comment:
Again, kindly ensure the Login screen, and the general design of your app is consistent with the [template design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=&), just as mentioned in previous reviews.
- [ ] Layout of each screen should match the original design. Items should be properly positioned, similar to the original design
- [ ] Contents should properly fit on mobile screen versions
- [ ] Ensure that typography (fonts) used matches the original design
- [ ] Background colors of items in a layout should match the original design
- [ ] A navigation button to move to and fro the screens should be present

Reply:
Done

Reply:
- [ ] Your app design still doesn't respect the [app design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=&). Please kindly make sure it respects the design.